src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -77,7 +87,21 @@ public Drivetrain() {
             DriveConstants.kRearRightTurningCanId,
             DriveConstants.kBackRightChassisAngularOffset);
 
+    // TODO: initialize this to where we place the robot on the field
```

Comment:
How?

Reply:
via auto chosen on smart dashboard, will default to default auto